src/pybind/mgr/balancer/module.py

```diff
@@ -1246,7 +1251,6 @@ def execute(self, plan):
 
         # upmap
         incdump = plan.inc.dump()
-        self.log.info("JR {0}".format(json.dumps(incdump)))
```

Comment:
@kamoltat we don't fix a fix in the same PR. could you squash the fix into the commit where the change to be reverted/amended was introduced?